src/adapter/src/client.rs

```diff
@@ -218,6 +227,28 @@ impl Drop for ConnClient {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ExtendedRequest {
+    #[serde(alias = "sql")]
+    /// 0 or 1 queries
```

Comment:
When is 0 acceptable? Can we make this always 1?

Reply:
The empty query `""`